modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/node/NodeUrl.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.cli.commands.node;
+
+import java.net.URL;
+
+/** Node URL. */
+public class NodeUrl {
```

Comment:
What's the purpose of this class? Can we simply use `URL` instead?

Reply:
We use it for cases when a parameter can be a URL or name. It has a special converter. And we aren't going to use that converter in cases when a parameter is just a URL. So we have to differentiate them.